deploy/k8s/chart/templates/kafka/020-PersistentVolumeClaim.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.deployPostgres }}
```

Comment:
Is the Kakfa PVC dependent upon deploying postgres?

Reply:
no … good catch